monday/query_joins.py

```diff
@@ -381,24 +460,71 @@ def get_boards_by_id_query(board_ids):
     }''' % board_ids
 
 
-def get_columns_by_board_query(board_ids):
-    return '''query
-        {
-            boards(ids: %s) {
+def duplicate_board_query(
+    board_id: int,
+    duplicate_type: DuplicateTypes,
+    board_name: str = None,
+    workspace_id: int = None,
+    folder_id: int = None,
+    keep_subscribers: bool = None,
+) -> str:
+    board_name = board_name if board_name else ""
+    workspace_id = workspace_id if workspace_id else None
+    folder_id = folder_id if folder_id else None
+    keep_subscribers = keep_subscribers if keep_subscribers else False
```

Comment:
For the 4 lines above, 
1. workspace_id and folder_id are already defaulting to None
2. why not set the defaults in the function definition board_name: str = "", keep_subscribers: bool = False ?
3. where possible, I'd avoid None since the typing is then incorrect (int = None will not be well liked); as we know that real workspace_id and folder_id will never be "0" why not set that as "falsy" value? Guess that's more of a preference

Reply:
1) agree
2) agree
3) Type annotations should be `int | None` or `Union[int, None]`

Then bellow: 

```
if workspace_id is not None:
    # do something

if folder_id is not None:
    # do something
```